### DISCUSSION TITLE
Ant Media Server 2.9.0 is available - Push Notification, OpenAPI, Easy Upgrade, ...

### DISCUSSION THREAD

**Discussion by mekya** [ant-media/Ant-Media-Server#6261] [Announcements]
Hi Guys,
We've released the 2.9.0 version. It's available on github for community edition and antmedia.io for enterprise edition. 

### Highlights
- Push Notification Support for Apple Push Notification and FirebaseCloud Messaging for EE - Create video/audio calls with Mobile SDKs
- Full OpenAPI support for REST Methods - Generate REST Methods SDKs(Java, JS, Python, Node, PHP) automatically
- Auto Start/Stop Stream Sources and IP Cameras. Start when there is a viewer, stops when there is no viewer
- Easily Upgrade server with `upgrade.sh` script when there is a new release 
- Improve Stability

[Full changelog is available here](https://github.com/ant-media/Ant-Media-Server/releases/tag/ams-v2.9.0)

**Comment by brixxnatt** [comment]
sorry how to update ?

**Comment by brixxnatt** [comment]
I have community edittion installed on linux server , please how to update without losing the data ?
thanks

**Comment by muratugureminoglu** [comment]
Hi @brixxnatt 

You can easily upgrade by following the document below.

https://antmedia.io/docs/guides/installing-on-linux/upgrading-ant-media-server/

**Comment by nathansegers** [comment]
Where can we find the documentation regarding the Push notifications?
Is this available in the Javascript SDK?

**Reply by mekya** [reply]
Hi @nathansegers,

Sorry, I missed your message. The documentation of the Push Notification is not available currently. It's in the pipeline.

Yes, it's available in JavaScript SDK. 

Fortunately, I'm happy to make a demo for you. If you're ok, let's schedule the meeting together.

I'm happy to connect with you on LinkedIn and I've sent the invitation to get connected ;)

Cheers
Oguz
